### PR TITLE
#366 Initial seed for IN1 to Coverage

### PR DESCRIPTION
**Pull request by cragun47**
Signed-off-by: Brian Cragun <cragun@us.ibm.com>

This is an initial seed for IN1.  It creates a small but FHIR valid Coverage object from key fields in IN1.
Also includes Class template.

@LisaWellman tag

### CODE REVIEW

File: src/main/resources/hl7/secondary/Class.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+---
+value: 
+     type: STRING
+     valueOf: $classValue
+     expressionType: HL7Spec
+     required: true
+
+type: 
+   valueOf: datatype/CodeableConcept_var
+   generateList: true
```
[the remark this answers is not in the document]
Comment (LisaWellman) [reply src/main/resources/hl7/secondary/Class.yml:15]: This is also required

**Review by LisaWellman — APPROVED** [review]
Looks good!

File: src/main/resources/hl7/resource/Coverage.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+#  Payor
+
+resourceType: Coverage
+id:
+  type: STRING
+  valueOf: 'UUID.randomUUID()'
+  expressionType: JEXL
+
+# Status is required, but it comes from a non-table 2 char ST 
+# It MUST be one of: active | cancelled | draft | entered-in-error; default to activew
```
[the remark this answers is not in the document]
Comment (cragun47) [reply src/main/resources/hl7/resource/Coverage.yml:19]: done

File: src/main/resources/hl7/secondary/Class.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (cragun47) [reply src/main/resources/hl7/secondary/Class.yml:15]: added.

File: src/main/resources/hl7/resource/Coverage.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+       orgIdentifier: String, IN1.3.1
+
+beneficiary:
+    valueOf: datatype/Reference
+    expressionType: resource
+    specs: $Patient
+
+class:
+   valueOf: secondary/Class
+   condition: $className NOT_NULL
```
[the remark this answers is not in the document]
Comment (cragun47) [reply src/main/resources/hl7/resource/Coverage.yml:44]: Done

**Pull request merged**